training/utils/dataset.py

```diff
@@ -50,13 +51,22 @@ def __len__(self):
         return len(self.image_list)
 
     def __getitem__(self, idx):
-        img_path = os.path.join(self.root_dir, 'annotations', self.image_list[idx])
-        target_path = os.path.join(self.root_dir, 'annotations', img_path.replace('.jpg', 'segcls.png'))
+        img_path = os.path.join(self.root_dir, 'RGB-PanSharpen', self.image_list[idx])
+        target_path = os.path.join(self.root_dir, 'annotations', self.image_list[idx].replace('.tif', 'segcls.tif'))
+
+        image = np.array(Image.open(img_path)).astype(np.int)
+        target = np.array(Image.open(target_path)).astype(np.int)
+        
+        # print('BEFORE converting target to building interior and border')
+        # print('Printing unique values of target: {}'.format(np.unique(target)))
 
-        image = np.array(Image.open(img_path))
-        target = np.array(Image.open(target_path))
         target[target == 100] = 1  # building interior
         target[target == 255] = 2  # border
+        target[target == 99] = 0 # Self Added by Vincent (Not sure what 99 is?)
+        target[target == 256] = 0 # Self Added by Vincent (Not sure what 99 is?)
```

Comment:
Not 100% sure about this. If I didn't do this, I get an error:

```
Traceback (most recent call last):
  File "training/train_aml.py", line 321, in <module>
    main()
  File "training/train_aml.py", line 292, in main
    step = train(loader_train, model, criterion, optimizer, epoch, step, logger_train)
  File "training/train_aml.py", line 166, in train
    loss = criterion(scores, y)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/module.py", line 550, in __call__
    result = self.forward(*input, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/loss.py", line 932, in forward
    ignore_index=self.ignore_index, reduction=self.reduction)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/functional.py", line 2317, in cross_entropy
    return nll_loss(log_softmax(input, 1), target, weight, None, ignore_index, None, reduction)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/functional.py", line 2117, in nll_loss
    ret = torch._C._nn.nll_loss2d(input, target, weight, _Reduction.get_enum(reduction), ignore_index)
IndexError: Target 99 is out of bounds.
```

I think it has something to do with annotations, but I'm not sure how to know what 100 vs 255 vs 99 vs 256 means?

Like in the original code, how did the author know that target == 100 corresponded to building interior?